app/build.gradle

```diff
@@ -47,15 +47,18 @@ android {
 
 dependencies {
     compile fileTree(include: ['*.jar'], dir: 'libs')
-    androidTestCompile('com.android.support.test.espresso:espresso-core:' + rootProject.espressoVersion, {
+    androidTestCompile('com.android.support.test.espresso:espresso-core:3.0.1', {
```

Comment:
keep using rootProject.espressoVersion and not the direct version number here.

Reply:
See? Merging is more work than doing it from scratch :smiley: